docs/source/AdministratorGuide/Tutorials/basicTutoSetup.sh

```diff
@@ -48,10 +36,28 @@ exec chpst -u dirac $RUNSVDIR -P /opt/dirac/startup 'log:  DIRAC runsv'
 # END runsvdir-start
 EOF
 
+sed -iE "s@ExecStart=/.*@ExecStart=/opt/dirac/sbin/runsvdir-start@g" /usr/lib/systemd/system/runsvdir-start.service
```

Comment:
Why did you chose to just replace the content of one line, instead of replacing the whole file, which is what is shown to the user, which might lead to inconsistent behaviour between the shell script and what the user should get from the tutorial?

Reply:
I didn't think of that. But yes, that makes more sense